cli/lib/kontena/cli/master/users/invite_command.rb

```diff
@@ -20,10 +21,12 @@ def execute
       else
         roles = []
       end
-
+      if email_list.size == 1 && self.external_id
+        external_id = self.external_id
+      end
       email_list.each do |email|
         begin
-          data = { email: email, response_type: 'invite' }
+          data = { email: email, external_id: external_id, response_type: 'invite' }
```

Comment:
Maybe check that it's really set before assigning it into the data. Also should this be only set for the first email, in case someone invites multiple emails and sets the external-id? Now this would try to set the same external id for all emails.